tidb-cloud/tidb-cloud-budget.md

```diff
@@ -0,0 +1,103 @@
+
+    > **Note:**
+    >
+    > If you are in multiple organizations, switch to your target organization by clicking its name.
+
+2. On the **Billing** page, click the **Budgets** tab.
+
+3. On the **Budgets** page, click **Create Custom Budget**. You can create up to five  custom budgets.
+
+4. Provide the budget basic settings.
+
+    - **Name**: enter a name for the budget.
+    - **Period**: select a time range for tracking costs. Currently, you can only select **Monthly**, which starts on the first day of each month and resets at the beginning of each month. TiDB Cloud tracks your actual spending during the time range against your budget amount (your planned spending).
+    - **Scope**: apply the scope to all projects (which means the entire TiDB Cloud organization) or a specific project as needed.
+
+5. Set the budget amount.
+
+    - **Budget Amount**: enter a planned spending amount for the selected period.
+    - **Apply credits and discounts**: choose whether to apply credits and discounts to the running total cost. Credits are used to reduce the cost of your TiDB Cloud usage, and discounts are reductions in the regular price of TiDB Cloud service. When this option is enabled, the budget tracks the running total cost minus credits and discounts.
+
+6. Configure alert thresholds for the budget. If your actual spending exceeds specified thresholds during the selected period, TiDB Cloud sends a budget notification email to your organization owners and billing administrators.
+
+    - By default, TiDB Cloud provides three alert thresholds: 75%, 90%, and 100% of the budget amount. You can modify these percentages as needed.
+    - To add a new alert threshold, click **Add alert threshold.**
+    - To remove a threshold, click the delete icon next to the threshold.
+
+7. Click **Create**.
+
+## Edit a custom budget
+
+> **Note:**
+>
+> The **Serverless Spending Limit** budget cannot be edited because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).
+
+To edit a custom budget, take the following steps:
+
+1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.
+
+    > **Note:**
+    >
+    > If you are in multiple organizations, switch to your target organization by clicking its name.
+
+2. On the **Billing** page, click the **Budgets** tab.
+
+3. On the **Budgets** page, locate the row of your budget, click **...** in that row, and then click **Edit**.
+
+## Delete a custom budget
+
+> **Note:**
+>
+> The **Serverless Spending Limit** budget cannot be removed because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).
```

Comment:
```suggestion
> The **Serverless Spending Limit** budget cannot be deleted because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).
```